manifests/vhost/aliases.pp

```diff
@@ -0,0 +1,20 @@
+
+define apache::vhost::aliases (
+  $aliases,
+  $vhost           = $name,
+) {
+  # determine fragment name
```

Comment:
This is a pattern you apply a lot. Would it make sense to define a function ```apache_get_fragment_name``` for this?